code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -751,6 +751,12 @@
 /obj/machinery/porta_turret/ai
 	faction = list("silicon")
 
+	var/emp_proofing = FALSE
+/obj/machinery/porta_turret/ai/emp_act(severity)
```

Comment:
weird formatting, move the var up a line and add a spacing

Reply:
Ahahaha... yeah... it's not like I spent 10 minutes or so trying to figure out what the concept of indentation is so I don't end up with 311 errors.. yeah.. I'll fix it

Reply:
Wasn't meant as a complaint also, I just found it hilarious how one or two misplaced tabs can cause hundreds of errors